py/desisim/templates.py

```diff
@@ -692,8 +730,8 @@ def make_galaxy_templates(self, nmodel=100, zrange=(0.6, 1.6), magrange=(21.0, 2
                     if nocontinuum or novdisp:
                         blurflux = restflux[this, :] * magnorm[this]
                     else:
-                        blurflux = (self.vdispblur((restflux[this, :] - thisemflux), vdisp[ii]) + \
-                                    thisemflux) * magnorm[this]
+                        blurflux = ((blurmatrix[blurindx[ii][0]][1] * (restflux[this, :] - thisemflux)) + \
```

Comment:
For long term maintainability, please add come comments to `_blurmatrix` explaining what the structure of `blurindx` and `blurmatrix` are.  The lists of tuples of (integer,matrix) ... make my head spin.  Could this structure be simplified?  This appears to be the only place that `blurmatrix` and `blurindx` are used, which would argue for simplifying the return structures of `_blurmatrix` so that the magic extra levels of `[0]` and `[1]` indexing wouldn't be needed here.  i.e. could you turn this into `blurmatrix[blurindx[ii]]`?

For consideration: if `blurmatrix` was a `dict` keyed by `vdisp[ii]` instead of a `list` indexed by `bluridx[ii]`, this would all boil down to `blurmatrix[vdisp[ii]]` which seems more understandable to me, and would simplify `_blurmatrix` too since you wouldn't need to do the `blurindx` bookkeeping.  Note: any concerns about two different values of vdisp being epsilon different would also apply to `list(set(vdisp))` in `_blurmaxtrix` so you're not avoiding that issue by using `blurindx`.